bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_generic.py

```diff
@@ -44,3 +44,8 @@ def pythonizegeneric(klass, name):
     AddPrettyPrintingPyz(klass)
 
     return True
+
+import cppyy
+cppyy.gbl.DeclareGenericCallable = GetCppCallableClass("Generic")
+cppyy.gbl.DeclareNumbaCallable = GetCppCallableClass("Numba")
+cppyy.gbl.DeclareCallable = GetCppCallableClass("Proxy")
```

Comment:
I am not sure `DeclareCallable` conveys the message of what the decorator does, since the decorated python function is already a callable. What about `CppCallable` ?

Then, for the numba option, we could have `NumbaCallable` or `CppCallable(numba = True)`. I am hesitating between the two.

Reply:
Done.